docs/topics/settings.rst

```diff
@@ -604,6 +604,21 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
```

Comment:
Maybe ``DOWNLOAD_FAIL_ON_DATALOSS``, which describes the current behavior: throw an error when ``ResponseFailed(reasons=[_DataLoss])`` happens (or any ``ResponseFailed`` error for that matter).